### ISSUE TITLE
NextDNS not properly resolving namebase TLDs

### ISSUE THREAD

**Issue by fabiogermann** [namebasehq/awesome-handshake#19]
In the "README.md" there is a reference to NextDNS to resolve the namebase domains.
However NextDNS does not properly resolve them - see:

- https://help.nextdns.io/t/y4h70bs/unable-to-resolve-handshake-domain#35haj40
- https://help.nextdns.io/t/60hg5y4/resolving-handshake-domains
- https://help.nextdns.io/t/y4yqxf6#m1yqx5c
- https://help.nextdns.io/t/p8y8497/handshake-namebase-web3-domains-routed-to-godaddy-aftermarket-sales

Can you clarify on the situation?

**Comment by Nathanwoodburn** [comment]
G'day,
Looks like it is something that NextDNS need to fix. In the meantime I'd recommend trying another DNS provider or using Fingertip

**Comment by fabiogermann** [comment]
@Nathanwoodburn I and others already raised it with them. No response.

I wonder if it would make a difference if you would contact NextDNS as an organization that promotes them.

Otherwise would you consider removing them as a recommendation? (To avoid confusion of new users?)

**Comment by Nathanwoodburn** [comment]
Yep. I'll send them a message but I'm not sure if it will help get them to add it.

**Comment by fabiogermann** [comment]
@Nathanwoodburn did you hear anything back from them?

**Comment by Nathanwoodburn** [comment]
Sorry nothing back from them yet.
